src/Sarif.Viewer.VisualStudio.Core/OutputWindowTracerListener.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.Sarif.Viewer
+{
+    public class OutputWindowTracerListener : TraceListener
+    {
+        private readonly string _name;
```

Comment:
>_ [](http://example.com/codeflow?start=32&length=1)

I'd prefer to stay way from this naming convention, partially because the Viewer codebase doesn't follow it.

Reply:
updated the naming.